### PR TITLE
Fix Mistral 7B v0.3 weight download

### LINKED ISSUE #1444: mistralai/Mistral-7B-v0.3 support
When will finetuning support for mistralai/Mistral-7B-v0.3 be integrated?

### PR DESCRIPTION
**Pull request by davmacario**
@rasbt Should be fine now, I hope! :)

I also had to re-add the model configs (litgpt/config.py), as they got deleted when you force-pushed. They should be correct (Mistral v0.3 uses a larger vocab size, but aside from that, all the settings are the same as for v0.2).

Closes #1444.

### CODE REVIEW

**Comment by davmacario** [comment]
Also, this should replace #1483

File: litgpt/config.py [hunk 1]
```diff
@@ -1530,6 +1530,42 @@ def norm_class(self) -> Type:
         intermediate_size=14336,
     )
 )
+configs.append(
+    # https://huggingface.co/mistralai/Mistral-7B-v0.3/blob/main/config.json
```
Comment (Andrei-Aksionov) [inline litgpt/config.py:1534]: From what repo this model is: unsloth or Mistral?
The link is to Mistral, but the org field in the config is unsloth.

Comment (davmacario) [reply litgpt/config.py:1534]: Just solved this in the new commit

File: litgpt/config.py [hunk 2]
```diff
@@ -1530,6 +1530,42 @@ def norm_class(self) -> Type:
...
+        norm_class_name="RMSNorm",
+        norm_eps=1e-05,
+        mlp_class_name="LLaMAMLP",
+        intermediate_size=14336,
+    )
+)
+configs.append(
+    # https://huggingface.co/mistralai/Mistral-7B-Instruct-v0.3/blob/main/config.json
+    dict(
+        name="Mistral-7B-Instruct-v0.2",
```
Comment (Andrei-Aksionov) [inline litgpt/config.py:1554]: It should be v0.3, right?

Comment (davmacario) [reply litgpt/config.py]: Yep, my bad

**Comment by rasbt** [comment]
Just tried it, and it works great! Thanks a lot for this contributions (and your patience)!

**Comment by davmacario** [comment]
No problem at all, I'm glad I could help!
And thank *you* for working on this project! It has been of great help for working on my MS thesis!

**Comment by rasbt** [comment]
Glad to hear it's useful, and all the best for the thesis!

**Pull request merged**